scripts/eqtl_hail_batch/generate_eqtl_spearman.py

```diff
@@ -195,7 +202,7 @@ def prepare_genotype_info(keys_path, expression_path):
         expression_df = pd.read_csv(AnyPath(expression_path), sep='\t')
         log_expression_df = get_log_expression(expression_df)
         mt = hl.read_matrix_table(TOB_WGS)
-        mt = hl.experimental.densify(mt)
+        # mt = hl.experimental.densify(mt)
```

Comment:
Can this be restored now, following [this fix](https://github.com/populationgenomics/hail/pull/206)?

Reply:
yes, sorry that as well is a holdover. I have now uncommented this